ansible/deploy_proxy.yml

```diff
@@ -125,48 +114,39 @@
   tasks:
     - include_role:
         name: nginx
-        tasks_from: custom_site
+        tasks_from: set_site_present
       vars:
+        site_present: "{{ 'cas_ssl' in special_sites }}"
         site_config: {name: cas_ssl, vars_file: cas_ssl}
-      when: "'cas_ssl' in special_sites"
 
 - name: Enikshay Proxy
   hosts: enikshay_proxy
```

Comment:
I still think that these can be incorporated into the `install other sites` task

Reply:
Just happened to see this and forget if I ever responded off-channel, but basically the thing that makes this not totally trivial is that these are run on hosts `enikshay_proxy`/`cas_proxy`/`pna_proxy` whereas the `install other sites` is  run on `hosts: proxy`. I didn't dig into it after noticing that; maybe there's enough of a system that this can be simplified and it just didn't happen to pop out at me.

Reply:
right that makes sense, thanks for the follow up